tcms/kiwi_auth/tests/test_backends.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase, modify_settings, override_settings
+from guardian.utils import get_anonymous_user
+
+
+@modify_settings(
+    AUTHENTICATION_BACKENDS={"append": "tcms.kiwi_auth.backends.AnonymousViewBackend"}
+)
+@override_settings(ANONYMOUS_USER_NAME="AnonymousUser")
```

Comment:
Had to do this because without this `AttributeError: 'Settings' object has no attribute 'ANONYMOUS_USER_NAME'` was raised

Reply:
This looks like a problem either with django-guardian or on our side b/c this setting isn't explicitly defined: https://github.com/django-guardian/django-guardian/issues/726

This override will make the test pass but the backend will still fail in production. For now explicitly define the setting in tcms/settings/common.py until we hear from upstream.